### DISCUSSION TITLE
closest(ray, point)

### DISCUSSION THREAD

**Discussion by OneLoneCoder** [OneLoneCoder/olcUTIL_Geometry2D#57] [General]
It had to happen of course! See #54 and #46. One appears to have an optimal implementation, and the other has some testing code. Of course we need both, so maybe its worth somehow mixing the two. Thoughts?

**Comment by Movinghead333** [comment]
I somehow overlooked that there was already a proposal for `closest(ray, point)`, my bad.

I have tested the code from #46 with my testing code and it works as expected. I would thus suggest to adopt the solution from #46 since it is way more concise (and most likely more performant) than the one I provided.

@meintte feel free to include the code for testing from my branch into yours. When #46 is merged, then I would just close my pull-request.
